### PR TITLE
The bookkeeping lesson updated

### PR DESCRIPTION
**Pull request by vlisovsk**
The bookkeeping path description was rather relevant for Run I samples, and is now updated.
Btw, the formulae in the challenges still do not work! So, the link to the DecFiles was moved out of the challenge box, so that it is still shown.

### CODE REVIEW

File: first-analysis-steps/bookkeeping.md [hunk 1]
```diff
@@ -59,14 +59,13 @@ not sure.
...
-We also have to choose the version of the
-digitisation and what configuration of the trigger (`Trig0x6138160F` in our case) and reconstruction
+use. There are two available in our case, `Sim09b` and `Sim09c`, but usually the latest available version is the best when there are more than one.
+We also have to choose what configuration of the trigger (`Trig0x6138160F` in our case) and reconstruction
 we want to have in the simulated sample. Usually
 there is only one choice for these, which makes choosing easier.
 
-We also have to select a version of the stripping. Choose any as long as it 
-contains the word `Flagged`.
+Two final steps are versions of `Turbo` (only for Run 2 samples, as discussed in the [previous lesson](run-2-data-flow.html)) and `Stripping`. To save the processing time, each of the Run 2 samples is processed via both Turbo and Stripping frameworks, so it is usable for either of two choices.
```
Comment (alexpearce) [inline first-analysis-steps/bookkeeping.md]: The motivation is more about making the samples as useful as possible for the largest number of analyses, rather than saving processing time.

File: first-analysis-steps/bookkeeping.md [hunk 2]
```diff
@@ -87,7 +86,7 @@ python file”, and add `.py` to the end of the text in “Save As...”. Clicki
 download. Save this file somewhere you can find it again.
 
 A copy of the file we just downloaded is [available
-here](data/MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09b_Trig0x6138160F_Reco16_Turbo03_Stripping28NoPrescalingFlagged_ALLSTREAMS.DST.py).
+here](data/MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09c_Trig0x6138160F_Reco16_Turbo03_Stripping28r1NoPrescalingFlagged_ALLSTREAMS.DST.py).
```
Comment (alexpearce) [inline first-analysis-steps/bookkeeping.md]: Seeing as we now have #146, is it necessary to download this file any more?

Comment (vbellee) [reply first-analysis-steps/bookkeeping.md]: Don't we need it so that we can interactively explore our DST?

File: first-analysis-steps/bookkeeping.md (hunk 1 above)
Comment (vlisovsk) [reply first-analysis-steps/bookkeeping.md]: Right, actually one probably doesn't save any processing time doing that :)
Let me rephrase this.

**Comment by chrisburr** [comment]
Thanks for all the work @vlisovsk! 👍
